source/atlas-cli-docker.txt

```diff
@@ -0,0 +1,63 @@
+.. _atlas-cli-docker:
+
+======================================
+Run {+atlas-cli+} Commands with Docker
+======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial shows you how to run {+atlas-cli+} commands with Docker
+in the following ways:
+
+- Run Docker in interactive mode
+- Run Docker as a daemon (detach mode)
+
+You can use interactive mode with or without an environment variable
+file.
```

Comment:
```suggestion
```
Nit (feel free to accept or reject): 

I don't think this is needed here, so I'd suggest removing it since it's covered in the `interactive mode` section